package.json

```
@@ -7,19 +7,17 @@
     "url": "https://github.com/storybookjs/jest.git"
   },
   "scripts": {
-    "build": "yarn build:esm && yarn build:cjs",
-    "build:esm": "tsc --project tsconfig.esm.json",
-    "build:cjs": "tsc --project tsconfig.cjs.json",
+    "build": "tsup ./src/index.ts --format cjs,esm --dts",
     "prerelease": "yarn build",
     "release": "auto shipit"
   },
-  "main": "./dist/cjs/index.js",
-  "module": "./dist/esm/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
```

shouldn't this be .mjs?
```suggestion
  "module": "./dist/index.mjs",
```